cmd/argocd/commands/admin/dashboard.go

```diff
@@ -42,7 +46,7 @@ $ argocd admin dashboard --port 8080 --address 127.0.0.1
 $ argocd admin dashboard --redis-compress gzip
   `,
 	}
-	initialize.InitCommand(cmd)
+	clientConfig = cli.AddKubectlFlagsToSet(cmd.Flags())
```

Comment:
Without passing the clientConfig directly from the cmd I was not able to use kubectl like flags to configure the cliebntConfig for eg. -n/--namespace was not working before this change. A similar issue was noticed by another [user](https://github.com/argoproj/argo-cd/issues/16266#issuecomment-1803864090)